pandas/tests/indexes/test_base.py

```diff
@@ -47,18 +46,6 @@
 class TestIndex(Base):
     _holder = Index
 
-    @pytest.fixture
-    def index(self, request):
-        """
-        Fixture for selectively parametrizing indices_dict via indirect parametrization
-        (parametrize over indices_dict keys with indirect=True). Defaults to string
-        index if no keys are provided.
-        """
-        key = getattr(request, "param", "string")
```

Comment:
This line makes that using `index` without parametrization is identical to writing `@pytest.mark.parametrize("indices", ["string"], indirect=True)`

Reply:
Thanks @SaturnFromTitan for doing this. This was intended to be a follow-up to #28865  https://github.com/pandas-dev/pandas/pull/28865#discussion_r333269878

Reply:
This is now replaced by `string_index`